src/Namshi/JOSE/JWT.php

```diff
@@ -34,9 +34,9 @@ class JWT
      */
     public function __construct(array $payload, array $header)
     {
-        $this->payload = $payload;
-        $this->header  = $header;
-        $this->encoder = new Base64UrlSafeEncoder();
+        $this->setPayload($payload);
```

Comment:
I think this is an artifact of the intermediary step where I had added a flag that you passed to turn off automatic claims inclusion which got backed out when switching to the subclass approach suggested by @odino. (That intermediary step has since been squashed+rebased out.) I can back this one out, though some would probably argue it is a good idea to use the public interface to prevent duplicated efforts if these setters ever do something more meaningful. Let me know and I'll fix it up.